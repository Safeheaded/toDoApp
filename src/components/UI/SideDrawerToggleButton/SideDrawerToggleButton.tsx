import React from 'react';
import { TouchableOpacity } from 'react-native';
import Icon from 'react-native-vector-icons/Ionicons';

// FIXME:
interface props {
    onToggle: () => void
}

const sideDrawerToggleButton = (props: props) => {

    return (
        <TouchableOpacity onPress={props.onToggle}>
            <Icon name="md-menu" size={35}/>
        </TouchableOpacity>
    );

}

export default sideDrawerToggleButton;