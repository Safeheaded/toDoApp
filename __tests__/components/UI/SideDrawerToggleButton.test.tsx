import React from 'react';
import {shallow} from 'enzyme';
import SideDrawerToggleButton from '../../../src/components/UI/SideDrawerToggleButton/SideDrawerToggleButton'

describe("Rendering", () => {
    it("Should match to snapshot", () => {
        const component = shallow(<SideDrawerToggleButton onToggle={() => {}} />);
        expect(component).toMatchSnapshot();
    })
});