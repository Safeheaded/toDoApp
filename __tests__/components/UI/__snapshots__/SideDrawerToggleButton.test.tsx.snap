// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering Should match to snapshot 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  onPress={[Function]}
>
  <Icon
    allowFontScaling={false}
    name="md-menu"
    size={35}
  />
</TouchableOpacity>
`;
